target_snowflake/__init__.py

```diff
@@ -430,6 +439,7 @@ def load_stream_batch(stream, records, row_count, db_sync, no_compression=False,
 
         # reset row count for the current stream
         row_count[stream] = 0
+        current_batch_size[stream] = 0
```

Comment:
Does this need to be done here if it's also being done directly in `persist_lines`, line 317?

Reply:
Line 317 is for initializing current_batch_size for the stream when schema line is received. Here we are resetting the current_batch_size to 0 after loading the batch.